my-react-app/src/Counter/Counter.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from "react";
 
 function Counter() {
-    const [count, setCount] = useState(0);
+    const [count,setCount ] = useState(0);
     const Increase = () => {
         setCount(count + 1);
```

Comment:
Don't use variable directly inside setter to update the state, use Pervious value which is provided by the callback in setter method